packages/api-plugin-discounts/src/util/setDiscountsOnCart.js

```diff
@@ -7,6 +7,9 @@ import getDiscountsTotalForCart from "../queries/getDiscountsTotalForCart.js";
  * @returns {undefined}
  */
 export default async function setDiscountsOnCart(context, cart) {
-  const { total } = await getDiscountsTotalForCart(context, cart);
-  cart.discount = total;
+  // check if promotion discounts are enabled
```

Comment:
Don't we want to remove this plugin altogether?

Reply:
> * In the `api-plugin-promotions` graphql schema the `PromotionFilter` input should be suffixed by `Input` to match the style of all graphql inputs.
> * The `createPromotion` mutation should use `PromotionInput` simple schema for the input. Also the following code in the same mutation:
> 
> ```
>   const { triggerKey } = promotions.triggers[0];
>   const trigger = promotions.triggers.find((tr) => tr.triggerKey === triggerKey);
>   promotion.triggerType = trigger.type;
> ```
> 
> Can't it be simplified to:
> 
> ```
> promotion.triggerType = promotions.triggers[0].type;
> ```

No, because it may be an array with multiple triggers at some point

Reply:
> Don't we want to remove this plugin altogether?

The plugin should be removed but I would like to retain backwards compatibility if possible